### ISSUE TITLE
Failed to start R, ioexception

### ISSUE THREAD

**Issue by lwahedi** [twosigma/beakerx#2615]
I get an error every time I try to start R: 

> stdout:
> java.io.IOException: Cannot run program "Rscript": CreateProcess error=2, The system cannot find the file specified 

A box also pops up saying: 

> Language Error
> Failed to start R.
> Did you install it according to the instructions on BeakerNotebook.com?
> If you already have it, then edit your preferences file to help Beaker find it on your system, and then restart Beaker and try again.
> Any other languages in your notebook should still work.

But my beaker.pref has the correct path to R.

> {
>  "languages" : {
>    "IPython" : {
>      "path" : "C:\Users\Name\Anaconda"
>    },
>    "R" : {
>      "path" : "C:\Program Files\R\R-3.2.1\bin\x64"
>    }
>  },
>  "pref-format" : "1",
> }

I've also tried the path ending in bin, which has an rscript.exe as well, but get the same error.

I am using Beaker version 1.4-0 and R 3.21

Any help would be much appreciated.

**Comment by scottdraves** [comment]
Thanks for the report.
You should not need to edit your prefs, as Beaker should be able to find R via a resource that the R installer sets (unless you disabled that).  How did you install R?
If you do need to set the path explicitly, then you should use double backslashes like this:

```
"path" : "C:\\Program Files\\R\\R-3.2.1\\bin\\x64"
```

Does that help?

**Comment by scottdraves** [comment]
Btw, I made a note of this in the wiki doc for the prefs: https://github.com/twosigma/beaker-notebook/wiki/Language-Preferences

**Comment by lwahedi** [comment]
I actually did try with the double backslashes and it didn't work. I specified the destination because I was using a previous install of R, though I did try before updating the prefs file, and it didn't work then either. I don't recall how I originally installed R, but the latest version was probably updated using installr. 

I did just get it to work, though. I used the installr package to update R to 3.2.2, and now it's working. My prefs are still as above, using the single backslashes. 

Thanks for the quick reply!
